src/main/java/SpringJPA/Model/UserType.java

```diff
@@ -0,0 +1,8 @@
+package SpringJPA.Model;
+
+public enum UserType {
+    TRIAL,
+    NONTRIAL,
+    PREMIUM,
```

Comment:
This should be changed to paid? Should premium not mean a special version of paid?

Reply:
As we don't have another version of a paid user there's no need to differentiate premium from paid.